<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>370</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Firebird Emu</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/resources/org.firebird-emus.firebird-emu.png</normaloff>:/icons/resources/org.firebird-emus.firebird-emu.png</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QToolButton" name="buttonReset">
             <property name="text">
              <string>Reset</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/resources/icons/system-reboot.png</normaloff>:/icons/resources/icons/system-reboot.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonPause">
             <property name="text">
              <string>Pause</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/resources/icons/Actions-media-playback-pause-icon.png</normaloff>:/icons/resources/icons/Actions-media-playback-pause-icon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonScreenshot">
             <property name="text">
              <string>Screenshot</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/resources/icons/camera-photo.png</normaloff>:/icons/resources/icons/camera-photo.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="buttonUSB">
             <property name="text">
              <string>Connect</string>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/icons/resources/icons/Actions-network-disconnect-icon.png</normaloff>
               <normalon>:/icons/resources/icons/Actions-network-connect-icon.png</normalon>:/icons/resources/icons/Actions-network-disconnect-icon.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonSpeed">
             <property name="toolTip">
              <string>Enable Fullspeed</string>
             </property>
             <property name="text">
              <string>Speed: 0 %</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="LCDWidget" name="lcdView" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tabDebugger">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/icons/tools-report-bug.png</normaloff>:/icons/resources/icons/tools-report-bug.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Debugger</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPlainTextEdit" name="debugConsole">
          <property name="font">
           <font>
            <family>Droid Sans Mono</family>
           </font>
          </property>
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ConsoleLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <family>Droid Sans Mono</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFiles">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/icons/document.png</normaloff>:/icons/resources/icons/document.png</iconset>
       </attribute>
       <attribute name="title">
        <string>File Transfer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="USBLinkTreeWidget" name="usblinkTree">
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <attribute name="headerDefaultSectionSize">
           <number>219</number>
          </attribute>
          <column>
           <property name="text">
            <string>Filename</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Size</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="refreshButton">
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/icons/resources/icons/system-reboot.png</normaloff>:/icons/resources/icons/system-reboot.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabSerial">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/icons/utilities-terminal.png</normaloff>:/icons/resources/icons/utilities-terminal.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Serial Monitor</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="serialConsole">
          <property name="undoRedoEnabled">
           <bool>false</bool>
          </property>
          <property name="lineWrapMode">
           <enum>QPlainTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/icons/resources/icons/preferences-desktop-keyboard.png</normaloff>:/icons/resources/icons/preferences-desktop-keyboard.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Keypad</string>
       </attribute>
       <layout class="QHBoxLayout" name="keypadLayout"/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEmulation">
    <property name="title">
     <string>&amp;Emulation</string>
    </property>
    <widget class="QMenu" name="menuRestart_with_Kit">
     <property name="title">
      <string>Restart &amp;with Kit</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/resources/icons/edit-bomb.png</normaloff>:/icons/resources/icons/edit-bomb.png</iconset>
     </property>
    </widget>
    <widget class="QMenu" name="menuBoot_Diags_with_Kit">
     <property name="title">
      <string>&amp;Boot Diags with Kit</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/icons/resources/icons/edit-bomb.png</normaloff>:/icons/resources/icons/edit-bomb.png</iconset>
     </property>
    </widget>
    <addaction name="actionReset"/>
    <addaction name="actionPause"/>
    <addaction name="actionRestart"/>
    <addaction name="menuRestart_with_Kit"/>
    <addaction name="menuBoot_Diags_with_Kit"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuFlash">
    <property name="title">
     <string>&amp;Flash</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionCreate_flash"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuLanguage">
     <property name="title">
      <string>Language</string>
     </property>
    </widget>
    <addaction name="actionScreenshot"/>
    <addaction name="actionRecord_GIF"/>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionLCD_Window"/>
    <addaction name="separator"/>
    <addaction name="actionDebugger"/>
    <addaction name="separator"/>
    <addaction name="actionXModem"/>
    <addaction name="actionSwitch_to_Mobile_UI"/>
    <addaction name="separator"/>
    <addaction name="menuLanguage"/>
   </widget>
   <widget class="QMenu" name="menuState">
    <property name="title">
     <string>S&amp;napshot</string>
    </property>
    <addaction name="actionSuspend"/>
    <addaction name="actionResume"/>
    <addaction name="separator"/>
    <addaction name="actionSuspend_to_file"/>
    <addaction name="actionResume_from_file"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
    <addaction name="actionAbout_Firebird"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuEmulation"/>
   <addaction name="menuTools"/>
   <addaction name="menuState"/>
   <addaction name="menuFlash"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/system-reboot.png</normaloff>:/icons/resources/icons/system-reboot.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Reset</string>
   </property>
  </action>
  <action name="actionDebugger">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/tools-report-bug.png</normaloff>:/icons/resources/icons/tools-report-bug.png</iconset>
   </property>
   <property name="text">
    <string>Enter &amp;Debugger</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/application-exit.png</normaloff>:/icons/resources/icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/Actions-media-playback-pause-icon.png</normaloff>:/icons/resources/icons/Actions-media-playback-pause-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="toolTip">
    <string>Pause execution</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/edit-bomb.png</normaloff>:/icons/resources/icons/edit-bomb.png</iconset>
   </property>
   <property name="text">
    <string>Re&amp;start</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/camera-photo.png</normaloff>:/icons/resources/icons/camera-photo.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Screenshot</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/Actions-network-disconnect-icon.png</normaloff>
     <normalon>:/icons/resources/icons/Actions-network-connect-icon.png</normalon>:/icons/resources/icons/Actions-network-disconnect-icon.png</iconset>
   </property>
   <property name="text">
    <string>Connect &amp;USB</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/media-floppy.png</normaloff>:/icons/resources/icons/media-floppy.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionCreate_flash">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/flash-create.png</normaloff>:/icons/resources/icons/flash-create.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Create Flash</string>
   </property>
  </action>
  <action name="actionXModem">
   <property name="text">
    <string>Send &amp;file over XModem</string>
   </property>
  </action>
  <action name="actionSuspend">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/system-suspend.png</normaloff>:/icons/resources/icons/system-suspend.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Suspend</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/system-suspend-hibernate.png</normaloff>:/icons/resources/icons/system-suspend-hibernate.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Resume</string>
   </property>
  </action>
  <action name="actionSuspend_to_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/system-suspend.png</normaloff>:/icons/resources/icons/system-suspend.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;to file</string>
   </property>
  </action>
  <action name="actionResume_from_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/system-suspend-hibernate.png</normaloff>:/icons/resources/icons/system-suspend-hibernate.png</iconset>
   </property>
   <property name="text">
    <string>Load &amp;from file</string>
   </property>
  </action>
  <action name="actionRecord_GIF">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/kmplayer.png</normaloff>:/icons/resources/icons/kmplayer.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Record GIF</string>
   </property>
  </action>
  <action name="actionAbout_Firebird">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/org.firebird-emus.firebird-emu.png</normaloff>:/icons/resources/org.firebird-emus.firebird-emu.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About Firebird</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionLCD_Window">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;External LCD</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/preferences-other.png</normaloff>:/icons/resources/icons/preferences-other.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Configuration</string>
   </property>
  </action>
  <action name="actionSwitch_to_Mobile_UI">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/resources/icons/smartphone.png</normaloff>:/icons/resources/icons/smartphone.png</iconset>
   </property>
   <property name="text">
    <string>Switch to Mobile UI</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LCDWidget</class>
   <extends>QWidget</extends>
   <header>lcdwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>USBLinkTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>usblinktreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>ConsoleLineEdit</class>
   <extends>QLineEdit</extends>
   <header>consolelineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
